CMakeLists.txt

```diff
@@ -21,20 +21,20 @@ project(Umpire
   LANGUAGES CXX C
   VERSION 0.3.2)
 
-set(ENABLE_OPENMP On CACHE Bool "")
```

Comment:
What's this change for?

Reply:
`BOOL` is the proper CMake variable type.
https://cmake.org/cmake/help/v3.3/command/set.html